revscoring/dependencies/dependent.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, name, process=None, depends_on=None,
         self.calls = 0
 
     def __call__(self, *args, **kwargs):
-        logger.debug("Executing {0}.".format(self))
+        logger.debug("Executing {0} ({1} calls so far)."
+                     .format(self, self.calls))
         self.calls += 1
```

Comment:
Since this is already inside a call, shouldn't it be swapted with `self.calls += 1` so that it says "1 calls so far" on first run?